### ISSUE TITLE
Make the android app running in background

### ISSUE THREAD

**Issue by sebastianovide** [soundscape-community/soundscape-web-client#33]
The current Android app behaves exactly like the web app, which means that when the phone is locked, it does not receive location updates. The goal is to enable background functionality so that you can put the phone in your pocket and the app will continue to work, similar to how Google Maps operates.

**Comment by sebastianovide** [comment]
mm.... cannot a simple way to update the compass and location and update the sounds accordingly while the phone is locked... There are few plugins for background gps but could not find any for the compass and spatial audio...

**Comment by steinbro** [comment]
Geolocation is the most important one, and I expect spatial audio would not be a separate plugin from background audio more generally.
 
FWIW, if we do want to have it work well in a pocket, we'd have to make the orientation detection a bit more sophisticated. I believe the original app detected when the user was moving, and if so assumed the user was facing the direction they were traveling. Right now, we rely solely on the device's physical orientation, but "pointing downward" will rarely be the intended direction

**Comment by sebastianovide** [comment]
apparently the iPhone users use the iPod so the orientation comes from them, rather than from the phone. That is what gives the user a real immersive experience feeling as if the sounds are coming from the real beacons.

**Comment by steinbro** [comment]
> apparently the iPhone users use the iPod so the orientation comes from them, rather than from the phone. That is what gives the user a real immersive experience feeling as if the sounds are coming from the real beacons

Yeah, you're talking about the head tracking built into some headphones like EarPods. I have no idea if that sensor data is exposed to a web app.

**Comment by sebastianovide** [comment]
yea... it looks like it is still under discussion... :thinking:  https://github.com/w3c/orientation-sensor/issues/68
